wasm/assembly/get_string.ts

```diff
@@ -1,3 +1,3 @@
-export function getString(): string {
-    return "hello test";
+export function getString(param: string): string {
```

Comment:
Could the example could be:

```ts
export function hello(name: string): string {
    return `"Hello, ${name}!`
}
```

Because when stuff is called `getString` or `params` it's hard to see at first sight if it's the name of a component of API exposed or the naming of an example item.

Reply:
@yvan-sraka I've just done a similar thing :wink: